scale_build/image/iso.py

```diff
@@ -45,8 +47,9 @@ def install_iso_packages_impl():
 
 
 def make_iso_file():
-    for f in glob.glob(os.path.join(RELEASE_DIR, '*.iso*')):
-        os.unlink(f)
+    if PRESERVE_ISO:
```

Comment:
The logic is inverted. If this is set to true, it will remove the ISOs but I think the exact opposite is wanted?

Reply:
Fixed